.github/workflows/build_and_test_debug_client.yml

```diff
@@ -102,6 +102,58 @@ jobs:
       - name: Build Linux Client
         run: npm run action client/electron/build linux
 
+  linux_arm64_debug_build:
+    name: Linux arm64 Debug Build
+    runs-on: ubuntu-24.04-arm
+    timeout-minutes: 20
+    needs: [web_test, backend_test]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+          cache: npm
+          cache-dependency-path: ./package-lock.json
+
+      - name: Install NPM Dependencies
+        run: |
+          sudo apt update && sudo apt install chromium-browser
+          npm ci
+
+      - name: Install Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: '${{ github.workspace }}/go.mod'
+
+      - name: Replace x86_64 OutlineProxyController
```

Comment:
This will also be removed in future versions (`deb`).

Reply:
That's ok. You can remove this step when OutlineProxyController is no longer needed